src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -108,6 +108,8 @@ class AuthScreens extends React.Component {
         UnreadIndicatorUpdater.listenForReportChanges();
         App.getAppData(false);
 
+        App.fixAccountAndReloadData();
```

Comment:
That's a good point and I think we should do this instead!

Reply:
So the reason I did it this way was because we wouldn't have to wait for User_FixAccount _most of the time_ anyway.

Since most of the time fixAccount is going to return `{changed: false}` even if it takes a negligible 50ms (or whatever the user's ping is at the time) to send/receive the response for fixAccount, that's extra time we'll ALWAYS have when waiting for this API command before we fetch the rest of the data.

On the flip side, we'd be making those 42 extra read requests _very rarely_ (pretty much only when we introduce a new migration). For all the times a user refreshes a page, or reopens an an app (not sure how it works on mobile), they'll be making so many more of those 42 read requests, that the rare 84 seems negligible.

I was thinking of it sort of as an optimistic action to just take the initial `App.getData()`.

Reply:
Ok, I can see how both sides of this argument can be right so probably doesn't matter right now.